### PR TITLE
Profile button

### PR DESCRIPTION
**Pull request by kaeleytran**
- profile button that displays name, field of work, and link to profile page of jobseeker passed through props 
- can archive, bookmark, or delete user 
- when a user is archived, can only unarchive or delete user and button changes color

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ function App() {
         <Route path="/dashboard/navigator" element={<NavigatorDashboard />} />
         <Route path="/reset" element={<Reset />} />
         <Route path="/profile" element={<Profile />} />
+        <Route path="/navigatordashboard" element={<NavigatorDashboard />} />
```
Comment (soliavalentine) [inline src/App.js]: can we remove this route? It routes to the same place as /dashboard/navigator

File: src/Components/navigatorDashboard.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+
+  var db = firebase;
+  const docRef = doc(db, "jobseekers", testemail);
+
+  useEffect(() => {
+    const logquery = async () => {
+      const query = await fetchJobseeker(testemail); 
+      // setJobSeeker({
+      //   name: query.data()["name"],
```
Comment (soliavalentine) [inline src/Components/navigatorDashboard.js]: we can remove this the jobseeker info is set in the assessment

File: src/Components/navigatorDashboard.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        </input>
+      </Modal>
+          </div>
+    </div>
+  );
+}
+else{
+  return (
+    <div>
+    {/* <Button onClick={onJobseekerClick} style={{ width: "100px", height:"50px",borderColor: 'grey', text:"client af" }} >jobseeker af</Button> */}
```
Comment (soliavalentine) [inline src/Components/navigatorDashboard.js]: please remove unused comments

File: src/Components/navigatorDashboard.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+    {/* {jobseeker.map(seeker => (
+            <h3>{seeker.name}</h3>
```
Comment (soliavalentine) [inline src/Components/navigatorDashboard.js]: unused comment?

File: src/Components/navigatorDashboard.js
Comment (soliavalentine) [inline src/Components/navigatorDashboard.js:1]: I think this component is virtually the same as kevin's navigatordashboard page if I'm not mistaken. If so I think we can remove this and just use the preexisting page!

File: src/Components/profileButton.js [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import './profileButton.css';
+// import Divider from '@mui/material/Divider';
+// import Paper from '@mui/material/Paper';
+// import MenuList from '@mui/material/MenuList';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+// import ListItemText from '@mui/material/ListItemText';
+// import ListItemIcon from '@mui/material/ListItemIcon';
```
Comment (soliavalentine) [inline src/Components/profileButton.js]: please remove unused comments

File: src/Components/profileButton.js
Comment (soliavalentine) [inline src/Components/profileButton.js:1]: When I test your component I don't see a difference in color when I attempt to archive, bookmark or delete. Is there a way I can test that functionality?

**Review by soliavalentine — CHANGES_REQUESTED** [review]
Great job Kaylee! This component looks really good and just needs a bit of cleanup to be ready to merge! Specifically:
- Remove unused comments/code and console.log statements
- Remove the navigator dashboard component since we ended up using kevins! (I think what may have happened is that you created a new branch off of your old branch rather than from main. In that case, I think that the easiest thing to do would be to pull from main and create a new branch and add your changes there.)
- I am a little unsure how to test your component. Maybe you could show me a demo with a jobseeker on monday!
-  I think the designers want there to be an icon to indicate whether or not a given jobseeker has been bookmarked, so it would be great if you could add this functionality!
- Lastly, since we want this component to be used for both jobseekers and navigators, we need to check if the user passed in is a navigator or jobseeker to see if we should display the archive feature!
I know this is a lot so please let me know if you have any questions, but once again great job!

File: src/Components/profileButton.js
Comment (kaeleytran) [reply src/Components/profileButton.js:1]: i think it should auto update???let me test again to make sure

**Pull request closed without merging**